glossary.yml

```diff
@@ -882,7 +882,15 @@
   en:
     term: "Anaconda"
     def: >
-      Anaconda is a [software distribution] (#software_distribution) of R and Python. It is also a [repository] (#repository) of open-source Python and R programs for data science, packaged using the [conda](#conda) [package manager] (#package_manager). Anaconda also creates Anaconda Navigator, a suite of desktop tools including an [IDE] (#ide) and the [Jupyter Notebook](#jupyter_notebook) application.  
+      Anaconda is a [software distribution] (#software_distribution) of R and Python. It is also a [repository] (#repository) of open-source Python and R programs for data science, packaged using the [conda](#conda) [package manager] (#package_manager). Anaconda also creates Anaconda Navigator, a suite of desktop tools including an [IDE] (#ide) and the [Jupyter Notebook](#jupyter_notebook) application.
```

Comment:
I am curious why the English text is shown as changed? I can't see visual differences. Better to avoid such situations - now `git blame` will show this in the history of this line

Reply:
Because of this line changed in both your branch and main (where apparently there was another commit changing it) we now have a conflict - you need to rebase your branch and resolve it before merging.